Boundary Docs (#2911)
hashicorp/web-unified-docs: content/boundary/v1.0.x/content/docs/deploy/terraform-patterns/terraform-targets.mdx

# END AUTO GENERATED METADATA
---

> # Terraform patterns for Boundary targets

Reviewer: ```suggestion
# Terraform patterns for targets
```

> Once you have defined a host, a host catalog, and a credential store, use Terraform patterns to create Boundary [targets](/boundary/docs/domain-model/targets) — the resource users connect to, with support for SSH and TCP protocols, injected or brokered credentials, and optional session recording.

Reviewer: ```suggestion
Once you have defined a host, a host catalog, and a credential store, use Terraform patterns to create Boundary [targets](/boundary/docs/domain-model/targets). Targets define the resources that users connect to and support SSH and TCP protocols, injected or brokered credentials, and optional session recording.
```
